src/Makefile

```diff
@@ -123,15 +123,14 @@ clean:
 	$(RM) metadatad $(OBJS)
 
 purge: clean
-	$(RM) -r init-submodules build-cpp-netlib build-yaml-cpp \
-	    $(CPP_NETLIB_DIR) $(YAML_CPP_DIR)
-	git submodule deinit --all
+	$(RM) -r init-submodules build-cpp-netlib build-yaml-cpp
+	git submodule deinit -f $(CPP_NETLIB_DIR) && \
```

Comment:
Do these commands need to be chained together? If so, can we indent the second line? Same elsewhere.

Reply:
I edited the existing Makefile, so have kept the same format.

I have updated the indents in the new Makefile though. PTAL.